packages/website/docusaurus.config.js

```diff
@@ -79,12 +79,7 @@ const config = {
             label: "Documentation",
           },
           { to: "/blog", label: "Blog", position: "left" },
```

Comment:
I think for the final presentation we want `Playground` to come to the left of `Blog`. But honestly I'm still not concerned `Blog` is valuable if it just CCs lesswrong posts -- less is more for the website. How about an `About` route that explains the value prop / summarizes parts of the lesswrong sequence / shows a use case from Nuño. We should also be thinking about quantifieduncertainty.org content! 

My user story is lesswrong/eaforum -> landing -> playground; _or_ landing -> about -> playground, and intuitively the order the routes appear in the top menu should reflect that, right? (genuinely uncertain-- I never learned a thing about this stuff)